src/main/java/software/amazon/smithy/lsp/SmithyLanguageServer.java

```diff
@@ -69,7 +69,7 @@ public CompletableFuture<InitializeResult> initialize(InitializeParams params) {
         loadSmithyBuild(workspaceRoot);
       } catch (Exception e) {
         LspLog.println("Failure trying to load extensions from workspace root: " + workspaceRoot.getAbsolutePath());
-        LspLog.println(e.toString());
+        e.printStackTrace();
```

Comment:
Does this create the same issue you described in [this comment](https://github.com/awslabs/smithy-language-server/pull/105/files#r1189255607)? Does printing to System.err mess with LSP transport?

Reply:
it does not, system.err is fine and gets piped to the output panel (at least in vscode-languageclient) :)
only stdin/stdout are a problem